testsuite/domain/src/test/resources/base-server-config.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this could be removed and used as a base for the domain-configs/domain-standard.xml which is indeed the base configuration used to start the domain for DomainTestSupport